pyphs/core/structure/connectors.py

```diff
@@ -35,8 +35,8 @@ def port2connector(core, i):
     moveCoreMcolnrow(core, core.dims.x()+core.dims.w()+i, core.dims.tot())
 
     # append port symbols to the list of connectors symbols
-    core.cu += [core.u[i], ]
-    core.cy += [core.y[i], ]
+    core.cu += core.Vector(core.u[i])
```

Comment:
`core.cu.append(core.u[i]) ` may be more readable, but I admit it is a matter of taste